### DISCUSSION TITLE
Update number of failed tests after rerun

### DISCUSSION THREAD

**Discussion by Maksbjj** [allure-framework/allure2#1720] [Questions & Support]
Hey guys 
Is it possible somehow to change the number of failed tests after tests rerun? I run java + selenium tests on gitlab, pass tests results as an artifacts from stage to stage and generate allure report in the very last stage using combined results from different test stages. The problem is that number of failed tests remain the same even when allure-results folder contain results from the rerun stage and some tests had passed on the second attempt. Is there any way to update this number or the only option is to get "updated" result for particular test in the retries tab? 
Thanks for any tips.

**Comment by cheshi-mantu** [comment]
@Maksbjj hi
do you have any parameters in your tests which change each time, e.g. hashes, timestamps whatsoever?
